content/blog/2023/10/camel-language-server-kubernetes-context/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "VSCode Language Support Autocompletion from Kubernetes Context"
```

Comment:
```suggestion
title: "VS Code Language Support Autocompletion from Kubernetes Context for secrets and configmaps"
```

there was already completion for namespace, so I think we need to be a bit more specific in the title.

VSCode --> VS Code